### PR TITLE
unblock 4.6 logging builds for dependencies

### PR DESCRIPTION
**Pull request by jcantrill**
Stop blocking logging bzs on release 4.7

### CODE REVIEW

**Comment by jcantrill** [comment]
@bparees does this do what I want it to, not block 4.6 logging repos on dependent issues?

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -677,6 +677,78 @@ bugzilla:
...
+              state_after_merge:
+                status: MODIFIED
+              state_after_validation:
+                status: POST
+              valid_states:
+              - status: NEW
+              - status: ASSIGNED
+              - status: ON_DEV
+            release-4.6:
+              dependent_bug_target_releases:
```
Comment (bparees) [inline core-services/prow/02_config/_plugins.yaml:700]: pretty sure the whole idea here is you do NOT want to require the bug to depend on another bug, when merging into the release-4.6 branch.  So i would assume you don't need this stanza at all.

Comment (jcantrill) [reply core-services/prow/02_config/_plugins.yaml:700]: @bparees the other examples I have seen they target the release, for example, to be 4.7 https://github.com/openshift/release/pull/17128/files#diff-a9035bbaee5af66130c81cfd0b97df589a7d2e22e87ad9d3800851407967c07bR626-L629

Comment (bparees) [reply core-services/prow/02_config/_plugins.yaml:700]: that link goes here

**Comment by hongkailiu** [comment]
Follow up https://coreos.slack.com/archives/CBN38N3MW/p1616683127280700

/lgtm

**Review by hongkailiu — APPROVED** [review]

**Comment by hongkailiu** [comment]
(some hiccups in Prow)

**Pull request merged**